src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -3293,20 +3312,21 @@ namespace HeatBalanceSurfaceManager {
         }
 
         // Beam and diffuse solar on inside surfaces from interior windows (for reporting)
-
-        for (SurfNum = 1; SurfNum <= TotSurfaces; ++SurfNum) {
-            if (!Surface(SurfNum).HeatTransSurf) continue;
-            //!!! Following may need to be removed or changed when shelves are considered in adjacent reflection calculations
-            if (Surface(SurfNum).Class == SurfaceClass_Shading) continue;
-            int const enclosureNum = Surface(SurfNum).SolarEnclIndex;
-            IntBmIncInsSurfIntensRep(SurfNum) =
-                ZoneBmSolFrIntWinsRep(enclosureNum) / DataViewFactorInformation::ZoneSolarInfo(enclosureNum).TotalSurfArea;
-            IntBmIncInsSurfAmountRep(SurfNum) = IntBmIncInsSurfIntensRep(SurfNum) * (Surface(SurfNum).Area + SurfaceWindow(SurfNum).DividerArea);
-            IntBmIncInsSurfAmountRepEnergy(SurfNum) = IntBmIncInsSurfAmountRep(SurfNum) * TimeStepZoneSec;
-            //      IntDifIncInsSurfIntensRep(SurfNum) = ZoneDifSolFrIntWinsRep(ZoneNum)/Zone(ZoneNum)%TotalSurfArea
-            //      IntDifIncInsSurfAmountRep(SurfNum) = IntDifIncInsSurfIntensRep(SurfNum) *  &
-            //                                             (Surface(SurfNum)%Area + SurfaceWindow(SurfNum)%DividerArea)
-            //      IntDifIncInsSurfAmountRepEnergy(SurfNum) = IntDifIncInsSurfAmountRep(SurfNum) * TimeStepZoneSec
+        if (SunIsUp) {
```

Comment:
Do any variables need to be set to 0 when the sun is down? I wonder if the last time these variables were calculated when the sun was up will be used during the night. Maybe they are set to 0 elsewhere.

Reply:
@rraustad There are 3 variables inside section:

                IntBmIncInsSurfIntensRep(SurfNum) 
                IntBmIncInsSurfAmountRep(SurfNum) 
                IntBmIncInsSurfAmountRepEnergy(SurfNum)

The variables are initialized at 0 in the beginning of InitSolarHeatGains before calling InitIntSolarDistribution. Therefore, no additional initialization is needed.